build.gradle

```diff
@@ -12,8 +14,36 @@ repositories {
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+    implementation("org.jetbrains:annotations:16.0.2")
+    implementation("org.jacoco:org.jacoco.core:0.8.7")
```

Comment:
Why do you add jacoco right into the application bundle? Does your application actually use jacoco inside?

Reply:
Removed